Description: don't check for keymap when using standalone
 This allows server to start without data content
Author: Quinton Reeves <qreeves@gmail.com>
Origin: upstream, http://redeclipse.net/svn/5504
Last-Update: 2013-03-26

--- a/src/engine/server.cpp
+++ b/src/engine/server.cpp
@@ -1407,11 +1407,13 @@
         chdir(dir);
     }
 #endif
+#ifndef STANDALONE
     loopi(3) if(!fileexists(findfile("data/config/keymap.cfg", "r"), "r"))
-    {
+    { // standalone solution to this is: pebkac
         if(i != 2) chdir("..");
         else fatal("could not find data directory");
     }
+#endif
     addpackagedir("data");
     defformatstring(gamedata)("game/%s", server::gameid());
     addpackagedir(gamedata);
